install/tools/setupIstioVM.sh

```diff
@@ -77,6 +77,7 @@ function istioInstall() {
 }
 
 function istioRestart() {
+    echo "restart istio" 
```

Comment:
Change to 
```
echo "*** Restarting istio proxy..."
```
for consitency with the other messages

Reply:
Done